### DISCUSSION TITLE
Request for a position to prompt in the object browser

### DISCUSSION THREAD

**Discussion by a4ginatl** [codefori/vscode-ibmi#2410] [Ideas]
I am working on a site with multiple LPARS, libraries, source files and members, courtesy of a vendor solution. There are over 250 source files in the main library and thousands of members.

It is difficult to locate and find source files and members. The filters don't help as there are too many options.

I would like to recommend adding a "position to" in the source files and members (2 steps) to help get to the required members more easily. This is in the object browser.
 
Adding a position to on the right click option would be a great help.

Thank you
Darryl Freinkel.

**Comment by worksofliam** [comment]
@a4ginatl there is a way to filter the list after it's loaded. The shortcut on Mac is command+option+f when focusing on the Object Browser.

I also believe there is a button to do this on the Object Browser as well.

**Comment by a4ginatl** [comment]
Is there a key sequence to use for Windows?

 

Darryl Freinkel

A4G

Telephone: 770.321.8562

**Reply by sebjulliand** [reply]
`ctrl+alt+f`

It is a standard VS Code feature available in every view (not specifically Code for IBM i's).

And that's the icon in the Browser:
![image](https://github.com/user-attachments/assets/2c657be8-8b2b-42dd-aea6-325fe1381326)

**Comment by a4ginatl** [comment]
Thanks, I found the “go to file” button. It worked.

 

Darryl Freinkel

A4G

Telephone: 770.321.8562
